README.md

```diff
@@ -68,3 +70,30 @@ Create Job Templates:
 Authenticate Ansible Tower to Automation-hub:
 - Retrieve token at https://cloud.redhat.com/ansible/automation-hub/token
 - Update token in Ansible Tower: https://www.ansible.com/blog/installing-and-using-collections-on-ansible-tower
+
+### OpenShift setup
+
+Additional Container Group on OpenShift
+- `oc create -f ocp-setup/role-pod-manager.yml`
+- `oc create -f ocp-setup/sa-tower-container-group.yml`
+- `oc create -f ocp-setup/rb-tower-container-group.yml`
```

Comment:
why not oneliner?

Reply:
This is just WIP documentation, I though this is more readable. 
In future I would like to code it all so that we can set it up from scratch on new OCP Cluster.

Reply:
even as oneliner you can use -f multiple times for one `oc` command IIRC